subgraphs/uniswap-v3/src/common/price/price.ts

```diff
@@ -171,7 +166,8 @@ export function findUSDPricePerToken(
             nativeTokenValueLocked.gt(largestNativeTokenValue) &&
             nativeTokenValueLocked.gt(
               NetworkConfigs.getMinimumLiquidityThreshold()
-            )
+            ) &&
+            token0.lastPriceUSD != BIGDECIMAL_ZERO
           ) {
             largestNativeTokenValue = nativeTokenValueLocked;
```

Comment:
Not in the scope of this PR, feel free to ignore.

`largestNativeTokenValue` doesn't seem to have any use?

Reply:
It does. It could use a better name though so I changed it. The priceSoFar only gets updated. When the whitelist token has a greater value than the previous pools.

Reply:
Ah right, I missed that `for` 🙈